.ci/travis/travis.sh

```diff
@@ -268,7 +268,7 @@ no-error-xwiki)
   # non-snapshot version until https://github.com/checkstyle/checkstyle/issues/9236
   # CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
   #                   --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
-  CS_POM_VERSION=8.39
+  CS_POM_VERSION=8.41-SNAPSHOT
```

Comment:
Please remove this line and uncomment previous

Reply:
Sorry, Its done.